erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -1243,25 +1243,28 @@ def set_status(self, update=False, status=None, update_modified=True):
 
 		precision = self.precision("outstanding_amount")
 		outstanding_amount = flt(self.outstanding_amount, precision)
+		due_date = getdate(self.due_date)
+		nowdate = getdate(nowdate())
```

Comment:
```suggestion
		nowdate = getdate()
```
`getdate` by default gives nowdate if noting is passed, no need to pass `nowdate()`